src/extension.ts

```diff
@@ -112,10 +112,20 @@ function downloadDeps(callback: Function) {
         let lines = str.split(/(\r?\n)/g);
         for (let i = 0; i < lines.length; ++i) {
             let l = lines[i];
-            if (l.startsWith('Got ')) {
-                let p = l.split(' ')[1];
-                if (!server) server = p;
-                else framework = p;
+            const serverPrefix = "Got language server: ";
+            const frameworkPrefix = "Got Checker Framework: ";
+            if (l.startsWith(serverPrefix)) {
+                const p = l.substring(serverPrefix.length);
+                if (!server) {    
```

Comment:
Now that this uses the full prefix, do we still need this check?
Should the framework branch below have similar checks?

Reply:
I think this isn't necessary. I will remove it.